app/views/layouts/_breadcrumbs_new.html.haml

```diff
@@ -0,0 +1,48 @@
+%span
+  %ol.breadcrumb.breadcrumbs
+    - if @title.nil?
+      - @title = ""
+    - if respond_to?(:data_for_breadcrumbs)
+      - @breadcrumb_new = data_for_breadcrumbs
+      - unless @breadcrumb_new.nil?
+        - if @right_cell_text.nil?
+          - @breadcrumb_new[0...-1].each do |breadcrumb_new|
+            %li
+              - if breadcrumb_new.key?(:url)
+                %a{:href => breadcrumb_new[:url], :onclick => "return miqCheckForChanges()"}
+                  = h(breadcrumb_new[:title])
+              - else
+                = h(breadcrumb_new[:title])
+          - if @title.end_with?("(Summary)") || @title.end_with?("(Dashboard)") || @title == @breadcrumb_new[-1][:title]
```

Comment:
I don't think you can do this: 1. it looks a bit heuristic, 2. `@title` is usually translated into current locale.